sql/hive/src/main/scala/org/apache/spark/sql/hive/client/ClientWrapper.scala

```diff
@@ -548,7 +547,15 @@ private[hive] class ClientWrapper(
   }
 
   def addJar(path: String): Unit = {
-    clientLoader.addJar(path)
+    val uri = new Path(path).toUri
```

Comment:
https://issues.apache.org/jira/browse/SPARK-16833, this isseue shows that sometimes it work. 
looks like sometimes `private val jars = new ConcurrentHashMap[String, File]()` in class NettyStreamManager doesn't successful add jar.